### PR TITLE
fix: Check contact url if is internal #250343

### PR DESCRIPTION
**Pull request by andreiggr**
fix: Added a check if urls in Contact component are internal or not. #250343

### CODE REVIEW

File: src/ui/Footer/Contact.jsx [hunk 1]
```diff
@@ -2,22 +2,49 @@ import React from 'react';
...
-                </Link>
+                <>
+                  {isInternalURL(child.link) ? (
+                    <Link to={child.link} key={index}>
+                      {child.text}
+                    </Link>
+                  ) : (
+                    <a
+                      href={child.link}
+                      title={child?.text}
```
Comment (ichim-david) [inline src/ui/Footer/Contact.jsx:40]: @andreiggr  there is no need to have title on the link

**Review by ichim-david — CHANGES_REQUESTED** [review]

File: src/ui/Footer/Contact.jsx (hunk 1 above)
Comment (andreiggr) [reply src/ui/Footer/Contact.jsx]: roger. Removed link in next commit

**Comment by ichim-david** [comment]
@eea-jenkins build storybook

**Comment by andreiggr** [comment]
@eea-jenkins build storybook

**Review by ichim-david — APPROVED** [review]

**Pull request merged**